data/newMigrations/001-programs.js

```diff
@@ -0,0 +1,15 @@
+exports.up = function (knex) {
+    return knex.schema.createTable('programs', tbl => {
+        tbl.increments('program_id');
+        tbl.text('program_name', 255)
+            .notNullable()
+            .unique();
+        tbl.text('program_description', 1000)
```

Comment:
You can remove the 1000, so `tbl.text('program_description')`, no need to add a max for a `text`